app/models/invoice.rb

```diff
@@ -1,4 +1,5 @@
 class Invoice < ApplicationRecord
+  default_scope { order('id ASC') }
```

Comment:
using a default scope can get tricky. this will most likely have unintended side effects. i'd suggest just calling this order clause wherever you need it for now 👍